sos/plugins/services.py

```diff
@@ -39,15 +36,4 @@ def setup(self):
         self.add_cmd_output("/sbin/chkconfig --list", root_symlink="chkconfig")
 
 
-class DebianServices(Services, DebianPlugin, UbuntuPlugin):
-
-    def setup(self):
-        super(DebianServices, self).setup()
-        self.add_copy_spec("/etc/rc*.d")
-
-        self.add_cmd_output("/sbin/initctl show-config",
-                            root_symlink="initctl")
-        if self.get_option('servicestatus'):
-            self.add_cmd_output("/sbin/initctl list")
-
```

Comment:
What was actually broken for Debian here? It's not clear from the patch description.

Reply:
Oops, nothing - made an assumption based on the /sbin/service 
- having said that this seems like it duplicates what is in upstart.py